src/App.js

```diff
@@ -0,0 +1,45 @@
+import './App.css';
+import { Link, Route, Routes } from 'react-router-dom';
+import MyProfile from './components/MyProfile';
+import Missions from './components/Missions';
+import Rockets from './components/Rockets';
+import planet from './planet.png';
+
+function App() {
+  return (
+    <>
+      <div className="wholeSiteContainer">
+        <nav>
+          <div className="navContainer">
+            <div className="logoHead">
+              <img src={planet} alt="planetImg" className="planet" />
+              <li className="heading">Space Travellers&apos; Hub</li>
+            </div>
+
+            <div className="linksContainer">
+              <li>
+                <Link to="/">Rockets</Link>
+              </li>
+              <li>
+                <Link to="/missions">Missions</Link>
+              </li>
+              <li>
+                <Link to="/MyProfile">MyProfile</Link>
+              </li>
+            </div>
```

Comment:
- [ ] Please use `NavLink` for navigation and ensure that only the active page is indicated with an underline as per requirements 👍
### Design Guideline
![image](https://github.com/bhushan354/Space-Travelers-Project/assets/46839548/85fda688-d5ce-4106-a95e-6672dc8710a7)
### Your Navigation
![image](https://github.com/bhushan354/Space-Travelers-Project/assets/46839548/7bab7185-b9ce-4ea0-b2ec-13b1622142d2)